app/components/Wallet/ReceiveModal/ReceiveModal.js

```diff
@@ -185,6 +190,7 @@ ReceiveModal.propTypes = {
   pubkey: PropTypes.string,
   address: PropTypes.string,
   alias: PropTypes.string,
+  activeWalletSettings: PropTypes.object,
```

Comment:
consider using shape here to make it clear what type of object we are expecting here

Reply:
done